nailgun/entities.py

```diff
@@ -6393,11 +6392,13 @@ class SyncPlan(
 
     def __init__(self, server_config=None, **kwargs):
         _check_for_value('organization', kwargs)
+        default_choices = ('hourly', 'daily', 'weekly')
         self._fields = {
             'description': entity_fields.StringField(),
             'enabled': entity_fields.BooleanField(required=True),
             'interval': entity_fields.StringField(
-                choices=('hourly', 'daily', 'weekly'),
+                choices=('hourly', 'daily', 'weekly', 'custom cron'),
+                default=gen_choice(default_choices),
```

Comment:
I don't understand why we assign random default value, isn't there a default value specified for the syncplan?

Reply:
when required and no default values a default will be chosen automatically from choices, in that case 'custom cron' can be also chosen, that require more options and this will make all the tests in robottelo to fail, also 'custom cron' is available for the moment only in upstream, that's why I recommend to to make explicitly default value to choose from the current values to preserve compatibility also.

Reply:
Ok, I thought hourly is the default, but probably just in the UI,  fair enough